Src/ToastNotifications/Position/ControlPositionProvider.cs

```diff
@@ -36,8 +36,21 @@ public Point GetPosition(double actualPopupWidth, double actualPopupHeight)
             if (source?.CompositionTarget == null)
                 return new Point(0, 0);
 
+            var elementSource = PresentationSource.FromVisual(_element);
+            if (elementSource == null)
+                return new Point(0, 0);
+
             Matrix transform = source.CompositionTarget.TransformFromDevice;
-            Point location = transform.Transform(_element.PointToScreen(new Point(0, 0)));
+            Point location;
+
+            try
+            {
+                location = transform.Transform(_element.PointToScreen(new Point(0, 0)));
+            }
+            catch (InvalidOperationException)
+            {
+                return new Point(0, 0);
+            }
 

```

Comment:
Here's the heart of the fix - ideally, lines 39-41 solve the problem and they are similar to the code right above it (35-37) which checks the source's CompositionTarget. However, since I can't reproduce this, I'm not 100% positive that the test on 39-40 will solve the problem, OR that the `_element` won't be disconnected by the time line 48 happens.

Considering that we see this problem often (maybe always?) when someone is logging in via Remote Desktop, I'm wondering if there's a race condition at play.